trollmoves/movers.py

```diff
@@ -30,7 +30,9 @@
 import sys
 import time
 import traceback
+import socket
 from ftplib import FTP, all_errors, error_perm
+from paramiko import SSHClient, SSHException
```

Comment:
So paramiko is not required in setup.py at the moment, which will cause the import of movers.py fail in a fresh install of trollmoves.
One possibility would be to add paramiko the requirements (as it is now only an extra).
However, if possible I would avoid it, because paramiko in not systematically needed (eg if using FTP).
And also @pnuu has been trying to reduce the number of dependencies...

Reply:
Did not think of that. Good point.